game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -65,6 +66,12 @@ class GameThread(scoreThread: ScoreThread) extends Runnable
     new Game(Game.RankedGameId, playerIDs, nextRankedDuration)
   }
 
+  def createPrivateGame(gameId: Int): Game = {
+    // We add the game's id since it's also the player's id at the same time
+    val playerIds = gameId :: ((-1 * Game.PrivateGameBotQuantity) to -1).toList
+    new Game(gameId, playerIds, Game.PrivateGameDuration)
```

Comment:
Well, I'm not 100% sure about that. I created an issue to check how to destroy the private games. For now, I think they will simply stay there forever